packages/state-transition/src/block/processExecutionLayerExit.ts

```diff
@@ -0,0 +1,50 @@
+import {byteArrayEquals} from "@chainsafe/ssz";
+import {electra} from "@lodestar/types";
+import {ETH1_ADDRESS_WITHDRAWAL_PREFIX, FAR_FUTURE_EPOCH} from "@lodestar/params";
+
+import {isActiveValidator} from "../util/index.js";
+import {CachedBeaconStateElectra} from "../types.js";
+import {initiateValidatorExit} from "./index.js";
+
+export function processExecutionLayerExit(state: CachedBeaconStateElectra, exit: electra.ExecutionLayerExit): void {
+  if (!isValidExecutionLayerExit(state, exit)) {
+    return;
+  }
+
+  const {epochCtx} = state;
+  const validatorIndex = epochCtx.getValidatorIndex(exit.validatorPubkey);
+  const validator = validatorIndex !== undefined ? state.validators.get(validatorIndex) : undefined;
```

Comment:
use `state.validaotrs.getReadonly()` for better performance, may not need to do it if `isValidExecutionLayerExit()` returns a `phase0.Validator`